src/main/java/uk/gov/pay/api/validation/MaxLengthValidator.java

```diff
@@ -3,7 +3,7 @@
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
 class MaxLengthValidator {
-    static boolean isValid(String value, int maxLength) {
-        return isBlank(value) || value.length() <= maxLength;
+    static boolean isInvalid(String value, int maxLength) {
+        return !isBlank(value) && value.length() > maxLength;
```

Comment:
```suggestion
        return isNotBlank(value) && value.length() > maxLength;
```
might make it more readable

Reply:
Makes sense - I let the IDE do the job.
I'll fix it in the follow-up PR.